_blog-src/_posts/2015-07-07-fuse-apis.markdown

```diff
@@ -0,0 +1,171 @@
+---
+layout: post
+title:  "Manage Fuse APIs with apiman"
+date:   2015-07-07 15:15:15
+author: Stefan Bunciak
+categories: api management jboss fuse
+---
+
+This article aims to provide a short guide on how to get API Management capabilities provided by apiman to work with JBoss Fuse, a lightweight, flexible, integration platform that is based on [Apache Camel](http://camel.apache.org), an implementation of many of the most commonly used enterprise integration patterns (EIP).
+
+<!--more-->
+
+Creating API provider in JBoss Fuse
+====================================
+
+The following few steps should quickly get you started with deploying applications into JBoss Fuse. For demonstration purposes I'm using a sample REST application (quickstart) shipped along with Fuse installation. For more information about developing applications for JBoss Fuse inspect [developer materials](http://www.jboss.org/products/fuse/developer-materials/#!project=fuse) and other [resources](http://www.jboss.org/products/fuse/resources/) at jboss.org.
+
+### Installation 
+To obtain a distribution of JBoss Fuse please visit the JBoss Fuse [download page](http://www.jboss.org/products/fuse/download/).
+Installation of JBoss Fuse is very simple, as the only thing you need to do is to unzip the distribution, e.g. 
+
+{% highlight sh %}
+unzip jboss-fuse-full-6.2.0.redhat-133.zip
+{% endhighlight %}
+
+You might want to also enable the default admin user by uncommenting the last line in `<fuse_dir>/etc/users.properties`. Without a user configured you won't be able to log into Fuse management console.
```

Comment:
- You might want to also enable 
- You might also want to enable

Grammatically okay, but a bit easier to understand.